packages/node-firebird-driver/src/test/tests.ts

```diff
@@ -273,22 +293,24 @@ export function runCommonTests(client: Client) {
 					// eventHandler may have a chance to cancel the events.
 					for (let i = 0; i < 20; ++i) {
 						await attachment.execute(transaction, `
-							execute block as
-							begin
-							    post_event 'EVENT1';
-							    post_event 'EVENT1';
-							    post_event 'EVENT2';
-							    post_event 'EVENT3';
-							end
-						`);
+              execute block as
```

Comment:
That changes is still weird.

I use following approach: indent with tabs up to the column where the command in string started. Then from there, indent with spaces. I also use VSCode and had no problem.

Please revert this.